src/api-reference/financial-integration/v4.financial-documents-schema.markdown

```diff
@@ -256,52 +256,52 @@ Name|Type|Format|Description
-`requestDate`|`Date`|yyyy-mm-ddThr:min:sec.msZ|Date of cash advance request from the detailed cash advance record.
-`requestedDisbursementDate`|`Date`|yyyy-mm-ddThr:min:sec.msZ|Cash advance disbursement date.
-`transactionType`|`Alphanumeric`|1 = Issue or Return to Administrator, 2 = Application, including Cash Advance Return expense within a report, or 3 = System Cash advance, from balance carry forward.|Type of transaction.
-`travelEndDate`|`Date`|yyyy-mm-ddThr:min:sec.msZ|The last day of the trip on the assigned travel request itinerary.
-`travelStartDate`|`Date`|yyyy-mm-ddThr:min:sec.msZ|The first day of the trip on the assigned travel request itinerary.
+`cardAccountID`|`String`|-|Alphanumeric. ID will be used initially by the receiving system to “condense” transactions associated with this card. It will also be used to retrieve the card number in a separate API call. Maximum length 32 bytes.
+`cardTransactionAmount`|`Number`|Maximum 23 digits to the left of the decimal; maximum 8 digits to the right of the decimal; with a maximum of 23 digits for the entire field.|Currency. Amount of the charge in the spend currency.
+`cardTransactionCurrency`|`String`|Alphanumeric. ISO 4217 3-letter alpha code.|Currency code for the spend currency.
+`cardTransactionID`|`String`|-|Alphanumeric. Calculated value assigned to this card entry during the import process. Maximum length 32 characters.
+`cardTransactionPostedAmount`|`Number`|Maximum 23 digits to the left of the decimal; maximum 8 digits to the right of the decimal; with a maximum of 23 digits for the entire field.|Currency. Amount of the charge in the billing currency of the card.
+`cardTransactionPostedCurrency`| `String`|Alphanumeric. ISO 4217 3-letter alpha code.|Currency code for the card billing currency.
+`cashAdvanceId`|`String`|-|GUUID. Unique system ID assigned to the cash advance. Maximum length 32 characters.
+`clearingAccountCode`|`String`|-|Alphanumeric. Contains the expense type account code. - or - If a CBCP Personal expense, the company card clearing account code. - or - If charge is tied to a Statement Report, and accounting code is set for Company Billed card account, the card's accounting code as specified in the **Account Code** field when creating or editing a CBS account. Maximum length 20 characters.
+`countryCode`|`String`|ISO 3166-1 alpha-2 country code.|Alphanumeric. The report country. Example: United States is US.
+`countrySubCode`|`String`|-|Alphanumeric. Report entry country sub code. Maximum length 6 characters.
+`currencyAlphaCode`|`String`|ISO 4217 3-letter alpha code.|Alphanumeric. Currency ISO alpha code for the spend currency if not an imported credit card or the invoice currency if this is a credit card.
```

Comment:
In some instances of the ISO code you include Alphanumeric is the Format and in others the Description. Either one works for me, just pick which you prefer so we can have consistency.